app/javascript/components/workflow-credential-mapping-form/helper.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useFieldApi, useFormApi } from '@@ddf';
+import MiqDataTable from '../miq-data-table';
+
+export const CredentialMapperComponent = (props) => {
+  const { rows, onCellClick } = useFieldApi(props);
+  const formOptions = useFormApi();
+
+  return (
+    <div className="credential-mapper-data-table">
+      <MiqDataTable
+        headers={[
+          { key: 'CredentialReferences', header: __('Credential Reference') },
+          { key: 'WorkflowCredentials', header: __('Workflow Credential') },
+          { key: 'CredentialField', header: __('Credential Field') },
+          { key: 'Delete', header: __('Delete') },
```

Comment:
Super-minor but I'm finding the phrase "Credential Reference" confusing here in the UI.  Maybe it's because under the covers the `credential_ref` is what's being called the "Workflow Credential" here?

Reply:
Or maybe "Credentials Identifier" (or "Workflow Identifier")

Reply:
Yeah I wasn't sure what to name each column in the table, could use some guidance.

Reply:
Yeah I think Credentials Identifier would be good here